seqr/utils/elasticsearch/es_utils_2_3_tests.py

```diff
@@ -1050,6 +798,19 @@ def _set_cache(k, v):
 MOCK_LIFTOVERS['hg19'].convert_coordinate.side_effect = lambda chrom, pos: [[chrom, pos + 10]]
 
 
+def sort_array(a):
+    sd={str(d):d for d in a}
+    return [sd[i] for i in sorted(sd.keys())]
+
+def sort_search(filter):
```

Comment:
Codacy found an issue: [Redefining built-in 'filter'](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5731912)